bgpd/rfapi/rfapi_vty.c

```diff
@@ -5178,7 +5178,7 @@ vnc_add_vrf_prefix (struct vty *vty,
 
 DEFUN (add_vrf_prefix_rd_label_pref,
        add_vrf_prefix_rd_label_pref_cmd,
-      "add vrf NAME prefix <A.B.C.D/M|X:X::X:X/M> [{rd ASN:nn_or_IP-address|label (0-1048575)|preference (0-4294967295)}]",
+      "add vrf NAME prefix <A.B.C.D/M|X:X::X:X/M> [{<rd ASN:nn_or_IP-address|label (0-1048575)|preference (0-4294967295)>}]",
```

Comment:
Thanks for the review Quentin.

Yes, using [{}] doesn't make any sense.

Does this look good to you?
```
add vrf NAME prefix <A.B.C.D/M|X:X::X:X/M> {<rd ASN:nn_or_IP-address|label (0-1048575)|preference (0-4294967295)>}
```

This version looks good to me but for some reason the "add vrf" command is not appearing in the output of "list permutations" anymore. Any idea of what might be wrong?